### ISSUE TITLE
stuck on "Loading..."

### ISSUE THREAD

**Issue by dimm0k** [syncthing/syncthing-android#756]
leaving syncthing running on Android has been a bit questionable in terms of performance. Upon starting syncthing it takes a few seconds to load before showing my list of shared folders, in which it then scans the two folders I have. if I leave the app running and don't touch my phone for a while (20 minutes about in this case), when I return to the phone syncthing is then stuck on "Loading..." I've tried leaving it in that state for about 5 minutes before clearing the app from the Recents list and opening the app again. at this point it opens up like nothing happened and I'm back to being able to use it again. during the loading I'm not able to pull any sort of log and opening it after there's nothing.
### Version Information

```
Syncthing Version: v0.14.8
Android Version: Android 6.0 Marshmallow / LGV10 / stock rooted
```

**Comment by Nutomic** [comment]
Can you try to get a log with `adb logcat`, or with a logcat app? One possibility is that Android killed Syncthing, so you can try to disable battery saving for the app. Also try the experimental settings.

**Comment by keksmampfer** [comment]
I have a similar problem. Syncthing doesn't start at all and got stuck at 'Loading...'. 

I was transferring a 2 gb file and multiple smaller files to the android device. The smaller files are synced with 5-7 mb/s but the big one started transferring wit 80 kb/s and then stopped completely. After a restart of syncthing it is just 'Loading ...'

Logcat shows this crash: http://pastebin.com/r7NcGwqJ

**Comment by Nutomic** [comment]
@keksmampfer This is a really weird crash. It mentions okhttp, but we dont use that. And it says there is a network call in the foreground thread, which doesnt make sense either. What version are you using, and how did you install it?

**Comment by capi** [comment]
@Nutomic the `com.android.okhttp` seems part of the internal implementation of `HttpUrlConnection`, which is used as far as I know.

**Comment by keksmampfer** [comment]
Tonight my Syncthing updated from 0.9.0 to 0.9.1 and it is now working correctly. The app loaded as usual. 
 
@dimm0k can you confirm?

**Comment by dimm0k** [comment]
i'm currently at version v0.14.13 on both Android and desktop and haven't been able to reproduce this issue since first reporting it, hence my silence in this ticket.

@keksmampfer

**Comment by Nutomic** [comment]
Ah this problem was actually fixed in 0.9.1. Due to a change, there could only be 256 API requests at the same time (and there can be many requests at once because we dont batch inotify updates).
